.github/scripts/trymerge.py

```diff
@@ -729,6 +729,7 @@ def merge_into(self, repo: GitRepo, *, force: bool = False, dry_run: bool = Fals
             self.merge_ghstack_into(repo, force, comment_id=comment_id)
 
         repo.push(self.default_branch(), dry_run)
+        gh_post_comment(self.org, self.project, self.pr_num, f"@{self.get_author()} your PR has been successfully merged.", dry_run)
```

Comment:
Can we just edit the last comment we posted? I feel like adding more comments like this is semi-annoying

Reply:
Agree with @suo's assessment over tracking timeline / generating notifications